### PR TITLE
Add __const to potential VTable Sections ##anal

### PR DESCRIPTION
**Pull request by thestr4ng3r**

### CODE REVIEW

**Comment by radare** [comment]
Screenshot 2019-07-01 at 12 08 31

File: libr/anal/vtable.c [hunk 1]
```diff
@@ -87,9 +87,10 @@ static bool vtable_section_can_contain_vtables(RVTableContext *context, RBinSect
 	if (section->is_segment) {
 		return false;
 	}
-	return !strcmp (section->name, ".rodata") ||
-		   !strcmp (section->name, ".rdata") ||
-		   !strcmp (section->name, ".data.rel.ro");
+	bool r = !strcmp (section->name, ".rodata") ||
+		!strcmp (section->name, ".rdata") ||
+		!strcmp (section->name, ".data.rel.ro") ||
+		r_str_endswith (section->name, "__const");
```
Comment (radare) [inline libr/anal/vtable.c]: missing return r; :PPP

Comment (thestr4ng3r) [reply libr/anal/vtable.c]: lol maybe I should actually compile my code before committing.

Comment (radare) [reply libr/anal/vtable.c]: :D

**Pull request merged**